integration-tests/metrics-v4.2.19-init.patch

```diff
@@ -0,0 +1,203 @@
+@@ -188,6 +188,7 @@ public class MetricsServlet extends HttpServlet {
+         return mapper.writer();
+     }
+ 
++    @SuppressWarnings("IdentityConversion")
+     protected TimeUnit parseTimeUnit(String value, TimeUnit defaultValue) {
+         try {
+             return TimeUnit.valueOf(String.valueOf(value).toUpperCase(Locale.US));
+diff --git a/metrics-servlets/src/main/java/com/codahale/metrics/servlets/MetricsServlet.java b/metrics-servlets/src/main/java/com/codahale/metrics/servlets/MetricsServlet.java
+index 0bd1297..7c69a08 100644
+--- a/metrics-servlets/src/main/java/com/codahale/metrics/servlets/MetricsServlet.java
++++ b/metrics-servlets/src/main/java/com/codahale/metrics/servlets/MetricsServlet.java
+@@ -189,6 +189,7 @@ public class MetricsServlet extends HttpServlet {
+         return mapper.writer();
+     }
+ 
++    @SuppressWarnings("IdentityConversion")
+     protected TimeUnit parseTimeUnit(String value, TimeUnit defaultValue) {
+         try {
+             return TimeUnit.valueOf(String.valueOf(value).toUpperCase(Locale.US));
+diff --git a/pom.xml b/pom.xml
+index 6192b52..bcbd571 100644
+--- a/pom.xml
++++ b/pom.xml
+@@ -68,7 +68,8 @@
+         <maven-compiler-plugin.version>3.11.0</maven-compiler-plugin.version>
+         <errorprone.version>2.19.1</errorprone.version>
+         <errorprone.javac.version>9+181-r4173-1</errorprone.javac.version>
+-
++        <error-prone-support.flags>-Xplugin:ErrorProne ${error-prone.flags}</error-prone-support.flags>
++        <java.version>8</java.version>
+         <sonar.projectKey>dropwizard_metrics</sonar.projectKey>
+         <sonar.organization>dropwizard</sonar.organization>
+         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+@@ -166,10 +167,9 @@
+                         <groupId>org.apache.maven.plugins</groupId>
+                         <artifactId>maven-compiler-plugin</artifactId>
+                         <configuration>
+-                            <compilerArgs>
++                            <compilerArgs combine.children="append">
+                                 <arg>-Xlint:all</arg>
+                                 <arg>-XDcompilePolicy=simple</arg>
+-                                <arg>-Xplugin:ErrorProne -XepExcludedPaths:.*/target/generated-sources/.*</arg>
+                                 <arg>-J--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED</arg>
+                                 <arg>-J--add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED</arg>
+                                 <arg>-J--add-exports=jdk.compiler/com.sun.tools.javac.main=ALL-UNNAMED</arg>
+@@ -181,18 +181,129 @@
+                                 <arg>-J--add-opens=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED</arg>
+                                 <arg>-J--add-opens=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED</arg>
+                             </compilerArgs>
+-                            <annotationProcessorPaths>
```

Comment:
It doesn't work if we leave this one here combined with the "append" usage in the other profile?

Reply:
Leaving this here results in a ` error: plug-in not found: ErrorProne`